templates/resources/tls_subscription.md.tmpl

```diff
@@ -112,14 +112,14 @@ In addition to the arguments listed above, the following attributes are exported
 * `created_at` - Timestamp (GMT) when the subscription was created.
 * `updated_at` - Timestamp (GMT) when the subscription was last updated.
 * `state` - The current state of the subscription. The list of possible states are: `pending`, `processing`, `issued`, and `renewing`.
-* `managed_dns_challenge` - The details required to configure DNS to respond to ACME DNS challenge in order to verify domain ownership. See Managed DNS Challenge below for details.
```

Comment:
Possibly only tangentially relevant to this PR, but it could be worth removing the "argument reference" and "attribute reference" sections from the docs template, as it gets added to the rendered docs page by `tfplugindocs` already

Reply:
Actually, I think we still have this problem: https://github.com/hashicorp/terraform-plugin-docs/issues/28
so I think we should leave it as-is for the time being.

Reply:
Ah yes good point